pkgs/development/tools/minipro/default.nix

```diff
@@ -0,0 +1,44 @@
+{stdenv, fetchgit, pkgs, pkg-config}:
+stdenv.mkDerivation rec {
+  name = "minipro-0.4";
+
+  src = fetchGit {
+    url = "https://gitlab.com/DavidGriffith/minipro.git";
+    rev = "896fe7d9d6f1e77931626d273e880856884ee6d8";
+  };
+
+  PKG_CONFIG = "${pkg-config}/bin/pkg-config";
+
+  nativeBuildInputs = with pkgs; [
```

Comment:
You shouldn't use `with pkgs;` here, import them at the top instead. You should drop the `pkgs` attribute all together.